.github/workflows/labeler.yml

```diff
@@ -44,23 +44,26 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: "3.11"
+      - name: Set up UV
+        run: curl -LsSf https://astral.sh/uv/install.sh | sh
       - name: Setup venv
         run: |
-          python -m venv venv
-          ./venv/bin/pip install -r hacking/pr_labeler/requirements.txt
+          uv venv venv
+          uv pip install --python venv \
+            -e hacking/pr_labeler -c tests/pr_labeler.txt
       - name: "Run the issue labeler"
         if: "github.event.issue || inputs.type == 'issue'"
         env:
           event_json: "${{ toJSON(github.event) }}"
           GITHUB_TOKEN: ${{ steps.create_token.outputs.token }}
           number: "${{ github.event.issue.number || inputs.number }}"
         run: |
-          ./venv/bin/python hacking/pr_labeler/label.py issue "${number}"
+          ./venv/bin/ad-triage issue "${number}"
```

Comment:
What does "ad" mean?

Reply:
ansible-documentation

Reply:
Oh, that wasn't obvious. Is it necessary to include project abbreviations in script names?

Reply:
Anyway, I'll mark this as resolved to unblock.